test/e2e/storage/persistent_volumes-local.go

```diff
@@ -356,6 +357,96 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("should support SELinux labeling on local storage volume", func() {
+
+		var testVol *localTestVolume
+
+		BeforeEach(func() {
+			testVol = setupLocalVolumePVCPV(config, node0)
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+		})
+
+		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2 when using the same SELinux label", func() {
+			By("Creating podSc1")
+			podSc1, podSc1Err := createSecPod(config, testVol, false, false, &v1.SELinuxOptions{
+				Level: "s0:c0,c1",
+			})
+			Expect(podSc1Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, podSc1))
+			podSc1NodeName, podSc1NodeNameErr := podNodeName(config, podSc1)
+			Expect(podSc1NodeNameErr).NotTo(HaveOccurred())
+			framework.Logf("Security Context POD %q created on Node %q", podSc1.Name, podSc1NodeName)
+			Expect(podSc1NodeName).To(Equal(node0.Name))
+
+			By("Creating podSc2")
+			podSc2, podSc2Err := createSecPod(config, testVol, false, false, &v1.SELinuxOptions{
+				Level: "s0:c0,c1",
+			})
+			Expect(podSc2Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, podSc1))
+			podSc2NodeName, podSc2NodeNameErr := podNodeName(config, podSc2)
+			Expect(podSc2NodeNameErr).NotTo(HaveOccurred())
+			framework.Logf("Security Context POD %q created on Node %q", podSc2.Name, podSc2NodeName)
+			Expect(podSc2NodeName).To(Equal(node0.Name))
+
+			writeCmd, readCmd := createWriteAndReadCmds(volumeDir, testFile, testVol.hostDir /*writeTestFileContent*/)
+			By("Writing in podSc1")
+			podRWCmdExec(podSc1, writeCmd)
+			By("Reading in podSc2")
+			readOut := podRWCmdExec(podSc2, readCmd)
+			Expect(readOut).To(ContainSubstring(testVol.hostDir)) /*aka writeTestFileContents*/
+
+			By("Deleting podSc1")
+			framework.DeletePodOrFail(config.client, config.ns, podSc1.Name)
+			By("Deleting podSc2")
+			framework.DeletePodOrFail(config.client, config.ns, podSc2.Name)
+		})
+
+		// This block should be run only when SELinux is enabled, otherwise it will fail on a negative test
+		if selinux.SELinuxEnabled() {
```

Comment:
Can you use Skipf() so that we log with the test framework that it's been skipped.